pkgs/development/libraries/science/math/tensorrt/extension.nix

```diff
@@ -38,13 +38,13 @@ final: prev: let
     + ".tar.gz";
 
   tensorRTVersions = {
-    "8.6.0" = [
+    "8.6.1" = [
       rec {
-        fileVersionCuda = "11.8";
-        fullVersion = "8.6.0.12";
-        sha256 = "sha256-wXMqEJPFerefoLaH8GG+Np5EnJwXeStmDzZj7Nj6e2M=";
+        fileVersionCuda = "12.0";
+        fullVersion = "8.6.1.6";
+        sha256 = "0f8157a5fc5329943b338b893591373350afa90ca81239cdadd7580cd1eba254";
```

Comment:
let's stick to using SRI hashes so that in the future we can migrate `sha256 =` -> `hash =` more easily

Reply:
Done !